src/services/k8s/knative.go

```diff
@@ -71,6 +74,62 @@ func loadManifest(project *project.Project, dockerImage docker.DockerImage) (*se
 	return service, nil
 }
 
+func loadEnvFile(envFile string) ([]corev1.EnvVar, error) {
+	var envVarList []corev1.EnvVar
+	if _, err := os.Stat(envFile); err != nil {
+		if os.IsNotExist(err) {
+			log.Info("No environment variables file (.env) to Load!")
+		} else {
+			return nil, fmt.Errorf("Error checking .env file: %v", err)
+		}
+	} else {
```

Comment:
If we pass the `envFile` as a parameter we can skip this and assume the file is there.

Reply:
I kept that in order to pick a default one & the user won't have to provide the flag if env var file is .env